### PR TITLE
Add windows group

### PR DESCRIPTION
**Pull request by spastorino**
r? @nikomatsakis

### CODE REVIEW

**Comment by nikomatsakis** [comment]
See https://github.com/rust-lang/compiler-team/issues/293 for context =)

**Comment by nikomatsakis** [comment]
@spastorino CI is unhappy?

File: teams/windows.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name = "windows"
+marker-team = true
+
+[people]
+leads = []
+members = [
+    "spastorino",
```
Comment (rylev) [inline teams/windows.toml:7]: Can you add the following people?
* @rylev 
* @danielframpton
* @sivadeilra 
* @lzybkr 
* @kennykerr
* @gdr-at-ms
* @luqmana
* @arlosi

Comment (retep998) [reply teams/windows.toml:7]: And me!

Comment (spastorino) [reply teams/windows.toml:16]: Done!.

File: people/sivadeilra.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name = 'sivadeilra'
```
Comment (rylev) [inline people/sivadeilra.toml:1]: This should be "Arlie Davis".

Comment (spastorino) [reply people/sivadeilra.toml]: Fixed it manually. `cargo run add-person` gets the name from Github.

**Pull request merged**